cloudmon/runner.py

```diff
@@ -84,11 +90,17 @@ def initialize_app(self, argv):
             final_config_dir = Path(self.config.private_data_dir, "_config")
             # final_config_dir.mkdir(parents=True, exist_ok=True)
             config_dir2 = None
+            if self.options.config_repo_branch is None:
```

Comment:
I would rather leave default from the type. Much less code. In addition to that you can set default on the argument itself and drop this if completely